### PR TITLE
Export sensitivity results as csv

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java [hunk 1]
```diff
@@ -129,6 +131,27 @@ public String getSensitivityAnalysisResult(UUID nodeUuid, String selector) {
...
+                .build()
+                .encode()
+                .toUri();
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        HttpEntity<SensitivityAnalysisCsvFileInfos> httpEntity = new HttpEntity<>(sensitivityAnalysisCsvFileInfos, headers);
+        return restTemplate.exchange(uri, HttpMethod.POST, httpEntity, byte[].class).getBody();
+
```
Comment (thangqp) [inline src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java:152]: To propagate exception as other methods

try {
           return restTemplate.exchange(uri, HttpMethod.POST, httpEntity, byte[].class).getBody();
} catch (HttpStatusCodeException e) {
           if (HttpStatus.NOT_FOUND.equals(e.getStatusCode())) {
                throw new StudyException(SENSITIVITY_ANALYSIS_NOT_FOUND);
            } else {
                throw handleHttpError(e, SENSITIVITY_ANALYSIS_ERROR);
            }
        }

Comment (YenguiSeddik) [reply src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java]: Done

File: src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java [hunk 2]
```diff
@@ -129,6 +131,35 @@ public String getSensitivityAnalysisResult(UUID nodeUuid, String selector) {
...
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        HttpEntity<SensitivityAnalysisCsvFileInfos> httpEntity = new HttpEntity<>(sensitivityAnalysisCsvFileInfos, headers);
+        try {
+            return restTemplate.exchange(uri, HttpMethod.POST, httpEntity, byte[].class).getBody();
+        } catch (HttpStatusCodeException e) {
+            if (HttpStatus.NOT_FOUND.equals(e.getStatusCode())) {
+                throw new StudyException(SENSITIVITY_ANALYSIS_NOT_FOUND);
+            } else {
+                throw handleHttpError(e, SENSITIVITY_ANALYSIS_ERROR);
```
Comment (thangqp) [inline src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java:157]: Using the current implementation of handleHttpError can not propagate exactly error message and returned code from the callee service => however, we should review handleHttpError later

**Review by thangqp — APPROVED** [review]
Code OK
Test OK

**Review by achour94 — APPROVED** [review]

**Pull request merged**